src/AerialShip/SamlSPBundle/Resources/doc/index.md

```diff
@@ -116,10 +111,11 @@ Now you have to tell to the Bundle what's your entity class
 # app/config/config.yml
 aerial_ship_saml_sp:
     driver: orm
-    sso_state_entity_class: Acme\SamlBundle\Entity\SSOState
-
+    sso_state_entity_class: AppBundle\Entity\SSOState
 ```
 
+If you are using a manager other then default, you can set it's name in ``model_manager_name`` config
```

Comment:
If you are using a manager other than default one, you can set it's name in ``model_manager_name`` config.

``` yaml
# app/config/config.yml
aerial_ship_saml_sp:
    model_manager_name: AppBundle\Entity\YourManager
```